numpy/core/tests/test_multiarray.py

```diff
@@ -4798,6 +4798,15 @@ def test_unicode_field_names(self):
             assert_raises(UnicodeEncodeError, np.dtype, [(nonencodable_name, int)])
             assert_raises(UnicodeEncodeError, np.dtype, [(('a', nonencodable_name), int)])
 
+            # A single name string provided to fromarrays() is also allowed to be unicode:
+            x = np.core.records.fromarrays([[0], [0]], names=u'a,b', formats=u'i4,i4')
```

Comment:
Is it worth checking the value / data structure of `x` produced or only that this doesn't raise an exception?